http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala

```diff
@@ -303,7 +306,7 @@ class ProxyService(
           // virtual UpgradeToWebSocket header which contains facilities to handle the WebSocket data.
           // The presence of this header distinguishes WebSocket from http requests.
           val res = for {
-            response <- request.header[UpgradeToWebSocket] match {
+            response <- request.attribute(AttributeKeys.webSocketUpgrade) match {
```

Comment:
I had thought attributes were different than headers? Is this an akka recommended change?